src/pages/organizers/create/OrganizerForm.tsx

```diff
@@ -111,7 +111,17 @@ const OrganizerForm = () => {
   // @ts-expect-error
   const organizer = getOrganizerByIdQuery?.data;
 
-  const organizerLabels = organizer?.labels;
+  const organizerLabelsSet = new Set<string>();
+
+  (organizer?.labels ?? []).map((label: string) =>
+    organizerLabelsSet.add(label),
+  );
+
+  (organizer?.hiddenLabels ?? []).map((label: string) =>
+    organizerLabelsSet.add(label),
+  );
```

Comment:
I would maybe build the Set by providing the arrays directly to avoid having to loop and add each entry separately:

```ts
const organizerLabelsSet = new Set(
    organizer?.labels ?? [], 
    organizer?.hiddenLabels ?? []
);
```

Should work equally in theory:

```js
new Set(['foo','bar'], ['foo'], ['bar'])
> Set [ "foo", "bar" ]
```

Reply:
I doesn't work since new Set(), only allows 1 argument. The 2nd argument will be ignored...
So e.g : 

```suggestion
new Set(['foo','bar'], ['foo'], ['hello'])
```

Will return
```suggestion
['foo','bar']
```
The "hello" is ignored...

Reply:
Yes sorry I should have tried with better dummy string 😅 but I would still do like `new Set([...a, ...b])` perhaps at least to avoid remaping the entries by one